modules/documentation/slides_documentation.md

```diff
@@ -0,0 +1,57 @@
+<!-- .slide: data-state="title" -->
```

Comment:
I feel like this overlaps with https://github.com/esciencecenter-digital-skills/research-software-support/pull/31/files#diff-4eda50d77350d14d82c95c621966f018914f5615010346089316abce43a8eca3 . I will comment more in the overall review.